JumboCode/jumbocode.org
# clients carousel

# Client carousel on about page

## Finished

- Infinitely rotating carousel with client logos
- Pro: It uses CSS animations so it's super cute and light
- Con: Keyframes in JS are gross
- Check out my idea to reduce color clutter!

## Still TODO
- Make logos link to actual client project pages
- Add more client logos

## To test

Import components/ClientCarousel to anywhere to see it in action. Or, see screen recording below.


https://user-images.githubusercontent.com/52360419/162845657-46a411cd-5906-4865-ac54-00841c0ee6db.mov
